docs/self_sign_cert.md

```diff
@@ -79,14 +79,14 @@ Now we save the updated keystore. We can either save to a new keystore and confi
 
 This option creates a _new_ list of CA Certificates and configures your build tool to use ths new list as the JRE's list of approved CA certificates, called the _trust store_.
 
-Within _KeyStore Explorer_, select _File > Save As..._ and save the new keystore file as a _JKS_ file within your project location. You will be prompted for a password; we use `password` in the examples below.
+Within _KeyStore Explorer_, select _File > Save As..._ and save the new trusted keystore file as a _JKS_ file within your project location. You will be prompted for a password; we use `password` in the examples below.
```

Comment:
we're still using the words keystore here. Should this be truststore file?

Reply:
Yeah, I think both of them are the keystore kind. So, the "trusted keystore" `cacerts` file. It's just that the convention is to use `cacerts.jks` and `keystore.jks`.

Reply:
Whaddya think about revising the file to use _truststore_ with an explanation at the top?  That would align with the JVM `javax.net.ssl.trustStore` property too.

Reply:
Yeah, I think changing most of "keystore" to "truststore" worked out.